index.html

```diff
@@ -1861,67 +1861,73 @@ <h3>Lifecycle Details</h3>
-          </li>
+        <figure id="life-cycle-details">
+          <img style="margin: auto; display: block; width: 85%;"
+         src="diagrams/ecosystemdetail.svg" alt="diagram showing how
+         credentials flow from issuer to holder, from and optionally 
+         from one holder to another; and how
+         presentations flow from holder to verifier, where
+         three parties can use information from a logical
+         verifiable data registry">
+          <figcaption style="text-align: center;">
+            The roles and information flows for this specification.
+          </figcaption>
+        </figure>
+
+        <ul>
           <li>
-The <a>issuer</a> issues a <a>verifiable credential</a> to the <a>holder</a>.
+An <a>issuer</a> <a>issues</a> a <a>verifiable credential</a> to a <a>holder</a>.
+Issuance always occurs before any other actions involving this credential. 
           </li>
           <li>
-The <a>holder</a> might pass on its <a>verifiable credentials</a> to another
-<a>holder</a>.
+A <a>holder</a> might transfer one or more of its <a>verifiable credentials</a>
+to another <a>holder</a>.
           </li>
           <li>
-The final <a>holder</a> presents its <a>verifiable credentials</a> to the
-<a>verifier</a> in a <a>verifiable presentation</a>, requesting a supported
-action.
+A <a>holder</a> presents one or more of its <a>verifiable credentials</a> to a
+<a>verifier</a>, optionally inside a <a>verifiable presentation</a>.
           </li>
           <li>
-The <a>verifier</a> <a>verifies</a> the authenticity of the
+A <a>verifier</a> <a>verifies</a> the authenticity of the
 <a>verifiable presentation</a> and <a>verifiable credentials</a>.
+This should include checking the <a>credential status</a> for revocation
+of the <a>verifiable credentials</a>.
           </li>
-          <li>
-The <a>verifier</a> <a>verifies</a> that the <a>holder</a> possesses the
-<a>verifiable credentials</a>.
-          </li>
-          <li>
-The <a>verifier</a> decides whether to accept the <a>verifiable credentials</a>
-for the requested action, taking into account its policy, the <a>holder</a>,
-and the contents of the <a>verifiable credentials</a>.
+         <li>
+An <a>issuer</a> may <a>revoke</a> a <a>verifiable credential</a>.
           </li>
           <li>
-The <a>verifier</a> either performs the requested action or rejects the
-request.
+A <a>holder</a> may delete a <a>verifiable credential</a>.
           </li>
-        </ol>
+        </ul>
 
         <p class="note">
-The order of the steps above is not fixed, and some steps might be taken
-more than once. Such step-recurrence may be immediate or at any later
-point in the sequence as performed.
+The order of the actions above is not fixed, and some actions might be taken
+more than once. Such action-recurrence may be immediate or at any later
+point.
         </p>
-
+        
+        <p>
+The most comon sequence of actions are envisioned to be
```

Comment:
`sequence of actions are envisioned`
-> `sequence of actions is envisioned`

Reply:
done. good catch.